package/skelcd-control-MicroOS.spec

```diff
@@ -67,108 +51,68 @@ Requires:       yast2-network >= 3.1.42
 # the other packages.
 Conflicts:      product_control
 Provides:       product_control
-
-# we really do not need the YaST packages for building, ignore the dependencies
-# to have faster builds and less rebuilds triggered by dependencies
-# these are pulled in by the skelcd-control-openSUSE dependencies
-#!BuildIgnore: yast2-caasp yast2-branding-openSUSE yast2-qt-branding-openSUSE
-#!BuildIgnore: autoyast2 yast2-add-on yast2-buildtools yast2-devtools
-#!BuildIgnore: yast2-fcoe-client yast2-firewall
-#!BuildIgnore: yast2-installation yast2-iscsi-client yast2-kdump yast2-multipath
-#!BuildIgnore: yast2-network yast2-nfs-client yast2-ntp-client yast2-proxy
-#!BuildIgnore: yast2-services-manager yast2-slp yast2-trans-stats yast2-tune
-#!BuildIgnore: yast2-update yast2-users yast2-x11 yast2-rdp
-#!BuildIgnore: yast2-reipl yast2-s390 yast2-vm
-#!BuildIgnore: rubygem(%{rb_default_ruby_abi}:byebug)
-#!BuildIgnore: yast2-branding-openSUSE-Oxygen yast2-configuration-management
-#!BuildIgnore: yast2-core yast2-hardware-detection yast2-installation-control
-#!BuildIgnore: yast2-logs yast2-perl-bindings yast2-pkg-bindings yast2-ruby-bindings
-#!BuildIgnore: yast2 yast2-ycp-ui-bindings
-
+Provides:       system-installation() = SMO
 # Architecture specific packages
 #
-%ifarch s390 s390x
-Requires:       yast2-reipl >= 3.1.4
-Requires:       yast2-s390
-%endif
-
-%ifarch %ix86 x86_64
+%ifarch %{ix86} x86_64
 Requires:       yast2-vm
 %endif
 
-#
-######################################################################
-
-Url:            https://github.com/yast/skelcd-control-MicroOS
-AutoReqProv:    off
-Version:        20200729
-Release:        0
-Summary:        The MicroOS control file needed for installation
-License:        MIT
-Group:          Metapackages
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
-Source:         %{name}-%{version}.tar.bz2
-
-%if 0%{?suse_version} >= 1500 && !0%{?skelcd_compat}
-%define skelcdpath /usr/lib/skelcd
-%endif
+# avoid file conflict with SLES package
+Obsoletes:	skelcd-control-leanos
 
 %description
-The package contains the MicroOS control file needed for installation.
+This package contains the control file used for SUSE MicroOS installation.
 
 %prep
 
-%setup -n %{name}-%{version}
+%setup -q -n skelcd-control-SMO-%{version}
 
 %build
-# build control.TWMicroOS.xml from control.MicroOS.xml
-make -C control control.TWMicroOS.xml
-# display the changes (just for easier debugging)
-# don't fail, a difference is expected
-diff -u control/control.MicroOS.xml control/control.TWMicroOS.xml || :
+%make_build -C control
 
 %check
-#
-# Verify syntax
-#
-make -C control check
+%make_build -C control check
 
 %install
 #
 # Add control file
 #
-mkdir -p %{buildroot}/%{?skelcdpath}/CD1
-install -m 644 control/control.TWMicroOS.xml %{buildroot}/%{?skelcdpath}/CD1/control.xml
+mkdir -p %{buildroot}%{_prefix}/lib/skelcd/CD1
+install -m 644 control/control.SMO.xml %{buildroot}%{_prefix}/lib/skelcd/CD1/control.xml
```

Comment:
those skelcd path looks better for me ( original ones )

Reply:
the name of the file was wrong indeed. %{?skelcdpath} is obsolete now I think. It is *not* a global macro and was required only for the transition before sle15 when we still had /CD1